src/exportCollection/index.ts

```diff
@@ -88,17 +88,17 @@ function getCollection(path): Promise<any> {
                 args.verbose && console.log(snap.ref.path);
 
                 // Decode Doc
-                decodeDoc(doc[snap.id]);
+                // decodeDoc(doc[snap.id]);
```

Comment:
Why did you comment this?

Reply:
this will make the CSV file too big to operate.